### PR TITLE
Adjustments to Phonebook Results Layout/Styles

### PR DESCRIPTION
**Pull request by cadie**
**Description**
Adjusts the phonebook results template, adding labels and slightly adjusting styles and spacing.

**Motivation and Context**
Adding labels helps with content organization and accessibility. Style and spacing adjustments assist in creating a better mobile experience and content hierarchy.

Individual notes:
- I moved the `.staff-list` style set so that they would be of higher importance than the `.phonebook-results` styles. This enabled the `.staff-list .name` font-weight and size styles to be used for the names in the staff list that do not have a `?query=` link applied
- Added `aria-labelledby` attributes in order to relate the "label" spans to the corresponding information.
- Now displays `N/A` if the phone number is empty. We could also add a wrapping if statement to the `.row` that contains the phone info, and I have that code stashed if that's preferred.
  - _You can see this by searching `psychology` in dev. The third result, "Psychology", has an empty phone number._

**How Has This Been Tested?**
Changes viewable in dev. 

**Types of changes**
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

**Checklist:**
- [x] My code follows the code style of this project.
- [ ] My change requires an update to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: includes/phonebook-functions.php [hunk 1]
```diff
@@ -323,25 +323,34 @@ function format_phonebook_result_primary( $result, $is_dept, $is_org, $is_group
...
-			<a href="https://map.ucf.edu/?show=<?php echo $result->bldg_id; ?>">
-				<?php echo phonebook_fix_name_case( $result->building ); ?>
-				<?php if ( $result->room ) : ?>
-					<?php echo ' - ' . $result->room; ?>
-				<?php endif; ?>
-			</a>
-		</span>
+		<div class="row mb-2">
+			<div class="col-3 col-md-12">
+				<span class="result-label text-default-aw text-uppercase" id="location-label">Location</span>
```
Comment (cjg89) [inline includes/phonebook-functions.php]: `location-label` ID could be repeated in phonebook results and be invalid.  Need to make sure the ID value here is always unique, or change the `aria-labelledby` on the location link to just use `aria-label="Location"` instead

File: includes/phonebook-functions.php [hunk 2]
```diff
@@ -323,25 +323,34 @@ function format_phonebook_result_primary( $result, $is_dept, $is_org, $is_group
...
+			</div>
+		</div>
 	<?php endif; ?>
 	<?php if ( $result->postal ) : ?>
-		<span class="postal d-block">
-			ZIP: <?php echo $result->postal; ?>
-		</span>
+	<div class="row mb-2">
+		<div class="col-3 col-md-12">
+			<span class="result-label text-default-aw text-uppercase" id="zip-label">Zip</span>
```
Comment (cjg89) [inline includes/phonebook-functions.php]: ZIP is an abbreviation/should be all-caps here

File: includes/phonebook-functions.php [hunk 3]
```diff
@@ -360,19 +369,43 @@ function format_phonebook_result_location( $result, $is_dept, $is_org, $is_group
...
-			Phone: N/A
-		</span>
-	<?php elseif ( $result->phone ) : ?>
-		<span class="phone d-block">
-			Phone: <a href="tel:<?php echo str_replace( '-', '', $result->phone ); ?>"><?php echo $result->phone; ?></a>
-		</span>
+	<?php if ( ! $is_group && $result->email ) : ?>
+		<div class="row mb-2">
+			<div class="col-3 col-md-12">
+				<span class="result-label text-default-aw text-uppercase" id="email-label">Email</span>
```
Comment (cjg89) [inline includes/phonebook-functions.php]: See comment above about repeating IDs

File: includes/phonebook-functions.php [hunk 4]
```diff
@@ -360,19 +369,43 @@ function format_phonebook_result_location( $result, $is_dept, $is_org, $is_group
...
+			<div class="col-9 col-md-12">
+				<span class="email">
+					<a href="mailto:<?php echo $result->email; ?>" aria-labelledby="email-label"><?php echo $result->email; ?></a>
+				</span>
+			</div>
+		</div>
 	<?php endif; ?>
+		<div class="row mb-2">
+			<div class="col-3 col-md-12">
+				<span class="result-label text-default-aw text-uppercase" id="phone-label">Phone</span>
```
Comment (cjg89) [inline includes/phonebook-functions.php]: See comment above about repeating IDs

File: includes/phonebook-functions.php [hunk 5]
```diff
@@ -360,19 +369,43 @@ function format_phonebook_result_location( $result, $is_dept, $is_org, $is_group
...
+				</span>
+			</div>
+		</div>
 	<?php if ( $is_group && $result->fax ) : ?>
-		<span class="fax d-block">
-			Fax: <?php echo $result->fax; ?>
-		</span>
+		<div class="row mb-2">
+			<div class="col-3 col-md-12">
+				<span class="result-label text-default-aw text-uppercase" id="fax-label">Fax</span>
```
Comment (cjg89) [inline includes/phonebook-functions.php]: See comment above about repeating IDs

File: src/scss/pages/_phonebook.scss [hunk 6]
```diff
@@ -1,3 +1,43 @@
...
+    }
+  }
+
+  .result-label {
+    font-size: 85%;
+  }
+
+  .email,
+  .phone {
+    text-overflow: ellipsis;
```
Comment (cjg89) [inline src/scss/pages/_phonebook.scss]: For `text-overflow` to work as expected, you'll need to force the text in these elements to overflow from their containers by adding:
```
overflow: hidden;
white-space: nowrap;
```

see https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow

Comment (cadie) [reply src/scss/pages/_phonebook.scss]: those styles were added along with `display: block;` to make that work correctly. Thanks!

**Pull request merged**